server/controllers/apply/applicationsController.ts

```diff
@@ -29,15 +29,17 @@ export default class ApplicationsController {
 
   show(): RequestHandler {
     return async (req: Request, res: Response, next: NextFunction) => {
-      const { application } = req.session
+      let application: Application
```

Comment:
I think you could do
`const application: Application =  req.session.application ?  req.session.application : await this.applicationService.findApplication(req.user.token, req.params.id)`

Reply:
Much better - love a one-liner